web/src/OverviewTable.tsx

```diff
@@ -568,23 +667,29 @@ export default function OverviewTable(props: OverviewTableProps) {
   } = useTable(
     {
       columns: columnDefs,
-      data,
+      data: props.data,
       autoResetSortBy: false,
     },
     useSortBy
   )
 
+  // TODO (lizz): since the structure of the table components has
+  // changed with groups, we should hide or show the widgets column
+  // globally and not on a per-table basis
+
   // if there are no widgets, hide the widgets column
   // TODO(matt) is this actually what we want?
-  const widgetsVisible = hasWidgets(props.view)
-  columns.forEach((c) => {
-    if (c.Header === "Widgets" && widgetsVisible !== c.isVisible) {
-      c.toggleHidden(!widgetsVisible)
-    }
-  })
+  // const widgetsVisible = hasWidgets(props.view)
+  // columns.forEach((c) => {
+  //   if (c.Header === "Widgets" && widgetsVisible !== c.isVisible) {
+  //     c.toggleHidden(!widgetsVisible)
+  //   }
+  // })
```

Comment:
I think it's fine to just delete this and worry about it if/when someone gives us a reason to